bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
> On 16 Dec 2015, at 10:36, Kai Kreuzer <notifications@github.com> wrote:
> 
> And is thus under the same licenses, so that we cannot add it to ESH.


Probably you are right - except that the dbus java library is acceptable for eclipse as they already agreed with the author that they can use? I just wondered if we should check this?


> They should then certainly also be part of the fragment. You will imho actually need different files for the different architectures - x86, x86_64, ARM, ARM64, etc.

Yes - this will be dependant on the system which is why the library is NOT in the bundle. It can be installed in linux with apt-get. Only the JAR was put into the bundle and it was expected that the user installed the system library themselves…

I’m happy to move this library to OH2 though - no problem.